docs/knowledge_base/issue_tags.md

```diff
@@ -0,0 +1,57 @@
+| IM0007    | NPM Hooks | Package runs the software immediately after installation |
+| HM0008    | [Typosquatting](https://docs.phylum.io/docs/typosquatting) | Package appears to be typosquatted |
+| CM0011    | [Hostname Detection](https://docs.phylum.io/docs/hostname_identification) | Package contains suspicious hostnames |
+| MM0012    | [Native Code](https://docs.phylum.io/docs/invokes_native_code) | Package contains calls used to load native code |
+| IM0013    | Dynamic Code | Package contains calls used to run dynamic classes |
+| ME0016    | [Secrets](https://docs.phylum.io/docs/secrets) | Secrets/tokens found in package not in test or example file |
+| IE0016    | [Secrets](https://docs.phylum.io/docs/secrets) | Secrets/tokens found in package in test or example file |
+| IM0017    | [Compiled Binaries](https://docs.phylum.io/docs/compiled_binary) | Package contains compiled binaries |
+| HM0018    | [Dependency Confusion](https://docs.phylum.io/docs/dependency_confusion) | Package has unusual semver or not found in registry |
+| IL0022    | License Mismatch | Package has a license mismatch between metadata and files |
+| HA0023    | [Ephemeral Author Domain](https://docs.phylum.io/docs/ephemeral_domain) | A disposable domain was used by a maintainer |
+| IE0023    | [IP Detection](https://docs.phylum.io/docs/ip_identification) | This package may contain hardcoded IP addresses. |
+| HM0023    | [Strange Python Imports](https://docs.phylum.io/docs/strange_python_imports) | Package imports things in a strange way |
+| CM0024    | [Remote Executable](https://docs.phylum.io/docs/remote_exe_ref_or_run) | Package runs remote executable |
+| MM0024    | [Remote Executable](https://docs.phylum.io/docs/remote_exe_ref_or_run) | Package references remote executable |
+| HM0025    | [Environment Variable Enumeration](https://docs.phylum.io/docs/env_var_enumeration) | Package enumerates through user folders |
+| IE0027    | [Trivial Package](https://docs.phylum.io/docs/trivial_package) | Package may be too small to be worth the security risk |
+| MM0028    | [Suspicious URL References](https://docs.phylum.io/docs/suspicious_url_references) | Package references pastebin-like sites |
+| HM0029    | [Obfuscated Python](https://docs.phylum.io/docs/obfuscated_python) | Package contains obfuscated Python |
+| .M0031    | [Suspicious Commands In Setup.py](https://docs.phylum.io/docs/suspicious_setup_commands) | Package contains unusual commands in setup.py |
+| HM0032    | [Exec on Remote URL](https://docs.phylum.io/docs/executes_code_at_remote_url) | Package calls `exec` on a remote URL |
+| HM0036    | [Webhook Exfil](https://docs.phylum.io/docs/webhook_exfil) | A package that appears to exfil data through a webhook |
+| HM0037    | Malware Bazaar Check | A package's file contains a hash that hits on Malware Bazaar |
+| CM0038    | Triaged Malware Rule (via threat feed) | Manually reviewed and confirmed to contain malware |
```

Comment:
```suggestion
| CM0038 | Triaged Malware Rule (via threat feed) | Manually reviewed and confirmed to contain malware |
| HM0099 | Obfuscated Javascript | Package contains obfuscated Javascript |
```

Reply:
I had this correct in my spreadsheet and wrote it incorrectly in the table. https://docs.google.com/spreadsheets/d/12bCbis85FmSSw4Nm9wYQtC3dEX8f-45xGj-xny45-eA/edit?usp=sharing

I'll check those one more time and make a PR to update the table in the rules repo.

Reply:
Thanks @spellman!